reana/reana_dev/git.py

```diff
@@ -29,6 +29,7 @@
     get_current_component_version_from_source_files,
     get_srcdir,
     is_feature_branch,
+    is_last_commit_release_commit,
```

Comment:
This one is defined I believe in #376, so this PR can wait until that one is merged.

Reply:
(... or move the function here if we are merging this first?!)